spack.yaml

```diff
@@ -43,6 +43,13 @@ spack:
   view: true
   concretizer:
     unify: true
+  modules:
+    default:
+      tcl:
+        all:
+          environment:
+            set:
```

Comment:
Would be good to add a comment along the lines of "DO NOT REMOVE! This environment variable must be set for the rAM3 suite to run correctly"